src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -22,9 +22,8 @@
 #include <ripple/app/ledger/InboundLedgers.h>
 #include <ripple/app/ledger/LedgerMaster.h>
 #include <ripple/app/misc/LoadFeeTrack.h>
-#include <ripple/core/ThreadEntry.h>
 #include <ripple/protocol/JsonFields.h>
-#include <ripple/beast/core/Thread.h>
+#include <ripple/beast/core/CurrentThreadName.h>
```

Comment:
I know this is settled, but I just wanted to add that I'm fine with what you have; thanks for investigating.